app/services/mastodon_tasks_service.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+class MastodonTasksService
```

Comment:
In my mind what is in `lib/` should not interacts directly with the application business logic and the best place I found was `services/` even if other stuff, here, are not the same.

About the name, `MaintenanceService`  ?